src/data/HomePage.yaml

```
@@ -53,51 +53,171 @@ tours:
 
 
 features:
-  - title: High-level Design
-    content: |
-      ## Yes
-      Some discussion about the feature and maybe a code block
-    code:
-
-  - title: Nested Hyperparameters
-    content: |
-      ## Some discussion of the feature
-      Bla Bla
+  - title: Choosing Models
+    content: >-
+      A Model Registry stores detailed metadata for [over 200 models](/machines) and
+      documentation can be searched without loading model code. 
     code: |
-      # 1. Load the Model
-      Tree = @iload DecisionTreeClassifier
-
-      # 2. Create an instance of the model
-      tree = Tree()
-
-      # 3. Wrap the model and data in a machine
-      mach = machine(tree, X, y)
-
-      # 4. Fit the machine
-      fit!(mach)
-
-      # 5. Make predictions
-      yhat = predict(mach, X)
-
-  - title: Neural Networks
-    content: Ding ding.
-    code: 
-
-  - title: Language Models
-    content: Cool
-    code:
+      # match models to data:
+      julia> X, y = @load_iris
+      julia> models(matching(X, y))
+      54-element Vector
+       (name = AdaBoostClassifier, package_name = MLJScikitLearnInterface, ... )
+       (name = AdaBoostStumpClassifier, package_name = DecisionTree, ... )
+       (name = BaggingClassifier, package_name = MLJScikitLearnInterface, ... )
+       ⋮
+      # filter by docstring content:
+      julia> models("pca")
+      (name = PCA, package_name = MultivariateStats, ... )
+      (name = PCADetector, package_name = OutlierDetectionPython, ... )
+
+      # retrieve docs without code import:
+      julia> doc("PCA", pkg="MultivariateStats")
+
+  - title: Meta-algorithms 
+    content: >-
+      For improved composability, and better data hygiene, an extensive number of
+      **meta-algorithms** are implemented as **model wrappers**:
+      <br><br>
+        - [Tuning](https://juliaai.github.io/MLJ.jl/dev/tuning_models/)
```

```suggestion
        - [Hyperparameter tuning](https://juliaai.github.io/MLJ.jl/dev/tuning_models/)
```
May be a reasonable suggestion given there are other types of tuning (e.g., fine-tuning in deep learning)